### DISCUSSION TITLE
miyoogamelist game names on Onion Launcher

### DISCUSSION THREAD

**Discussion by Artickw** [OnionUI/Onion#533] [Ideas]
Hi!

Is it possible to make Onion Launcher show the names on miyoogamelist.xml instead of the filenames? I've manually added the miyoogamelist.xml on all of my current consoles, and it shows the names the way I want in the menu, but not on Onion Launcher. Is there something that I can do about it or can it be added on a future update?
Thanks for your hard work!

**Comment by Sichroteph** [comment]
Thanks for your feedback it is on the to do list!
